src/components/dialogs/shunt-compensator-creation-dialog.js

```diff
@@ -149,6 +180,70 @@ const ShuntCompensatorCreationDialog = ({
         }
     };
 
+    const handleSelectionChange = (element) => {
+        let msg;
+        fetchShuntCompensatorInfos(
+            studyUuid,
+            selectedNodeUuid,
+            element.id
+        ).then((response) => {
+            if (response.status === 200) {
+                response.json().then((shuntCompensator) => {
+                    setFormValues(null);
```

Comment:
same remark in all dialogs ... is this line usefull, as formValues is set just after ...